user_manual/docs/main.md

```diff
@@ -3,10 +3,56 @@ Introduction     {#mainpage}
 
 [TOC]
 
-Explain what it is here.
+This guide describes how to get started with a device running the
+[Open Lighting Embedded](https://www.openlighting.org/ole/) (OLE) software. For
+developer information, including instructions on how to modify the OLE code,
+see the [OLE Developer Guide](http://docs.openlighting.org/ja-rule/doc/latest/).
 
-Explain the relationship between the code and the device (maybe punt this to
-later since people will just want to get it working).
+Since the OLE software is customizable, the version running on a particular
+device may have different functionality from what is described here. Consult
+your manufacturer's documentation on the specific features enabled on a
+particular hardware product.
```

Comment:
Software perhaps then. You first say the software is customisable, then you say ask your manufacturer about what your particular hardware enables. They may have a Number 1 with custom firmware on it, loaded by someone else.